node/src/cli.rs

```diff
@@ -42,6 +42,9 @@ pub enum Subcommand {
     Benchmark(frame_benchmarking_cli::BenchmarkCmd),
 
     /// Decodes a base58 PeerID and returns its hex and bytes forms
-    #[structopt(name = "decode-peer-id", about = "Provides Hex version of base58 PeerId.")]
-    PeerIdHex(supra_subcommands::PeerIdHexCmd)
+    #[structopt(
+    name = "decode-public-key",
+    about = "Decodes a Provided Public Key to generate."
```

Comment:
Can you change the about - to `about = "Decodes a Public Key to generate PeerID"`